release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1277,6 +1277,47 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
+
+* Previously, the version number text in the *Updates* graph on the *Cluster Settings* page appeared as black text on a dark background when using Firefox in dark mode. With this update, the text appears as white text. (link:https://issues.redhat.com/browse/OCPBUGS-38427[*OCPBUGS-38427*])
+
+* Previously, the *Cluster overview* page included a `View all steps in documentation` link that resulted in a 404 error for ROSA and OSD clusters. With this update, the link does not appear for ROSA and OSD clusters. (link:https://issues.redhat.com/browse/OCPBUGS-37054[*OCPBUGS-37054*])
+
+* Previously, dynamic plugins using PatternFly 4 were referencing variables that are not available in {product-title} 4.15 and later. This was causing contrast issues for ACM in dark mode. With this update, older chart styles are now available to support PatternFly 4 charts used by dynamic plugins. (link:https://issues.redhat.com/browse/OCPBUGS-36816[*OCPBUGS-36816*])
+
+* Previously, when the `Display Admission Webhook` warning implementation presented issues with some incorrect code. With this update, the unnecessary warning message has been removed. (link:https://issues.redhat.com/browse/OCPBUGS-35940[*OCPBUGS-35940*])
+
+* Previously, the global sync lock that applied to all HTTP-serving spawned gorotines with a sync lock that is specific to each of the refresh tokens. With this update, the global refresh sync lock on a cluster with an external OIDC environment was replaced with per-refresh token one. As a result, refresh token performance is improved by 30 to 50%. (link:https://issues.redhat.com/browse/OCPBUGS-35080[*OCPBUGS-35080*])
+
+* Previously, a warning was not displayed for `minAvailable` in `PodDisruptionBudget` create and edit form. With this update, code logic for displaying `minAvailable` warning was added, and the `minAvailable` warning is displayed if violated. (link:https://issues.redhat.com/browse/OCPBUGS-34937[*OCPBUGS-34937*])
+
+* Previously, the *OperandDetails* page displayed information for the first CRD that matched by name. After this fix, the *OperandDetails* page displays information for the CRD that matches by name and the version of the operand. (link:https://issues.redhat.com/browse/OCPBUGS-34901[*OCPBUGS-34901*])
+
+* Starting with 4.16 release, Console provided PatternFly 4.x shared modules are deprecated and will be removed in the future. console plugins are encouraged to upgrade to PatternFly 5. Also starting with 4.16 release, React Router v5 shared modules that were provided are now deprecated and will be removed in the future.
```

Comment:
```suggestion
* Starting with the 4.16 release, console-provided PatternFly 4.x shared modules are deprecated and will be removed in the future. You are encouraged to upgrade your console plugins to PatternFly 5. Also starting with the 4.16 release, React Router v5 shared modules that were provided are now deprecated and will be removed in the future.
```

Reply:
This doesn't actually belong here so I'm removing it.